deployment/ansible/roles/driver.app/templates/nginx-app.conf.j2

```diff
@@ -13,7 +13,7 @@ server {
         proxy_set_header X-Forwarded-Server $host;
         proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
 
-        try_files $uri $uri/ /index.html;
```

Comment:
Do all the app pages still work with this change? After replying to #84 I was told that a fallback to `/index.html` is required when `html5mode = true` in order to allow Angular to handle routing paths that nginx can't find. I'd love it if that weren't true though since I think that recursively redirecting forever is a poor substitute for 404s.